### PR TITLE
cordova-config 0.0.1

### PR DESCRIPTION
**Pull request by blackjid**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/cordova-config/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+title: cordova-config
```
Comment (godrei) [inline steps/cordova-config/0.0.1/step.yml]: The step's title is a user facing name of the step, i would recommend to use: `Manipulate cordova config.xml` for example

File: steps/cordova-config/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    is_required: false
+    summary: Modify the attribute `android-versionCode` in the config.xml file
+    title: Android version code
+- android_packageName: $CORDOVA_ANDROID_PACKAGENAME
+  opts:
+    is_expand: true
+    is_required: false
+    summary: Modify the attribute `android-packageName` in the config.xml file
+    title: Android package name
+- ios_CFBundleVersion: $CORDOVA_IOS_BUNDLEVERSION
```
Comment (godrei) [inline steps/cordova-config/0.0.1/step.yml]: please use snake_case input keys, like: `ios_bundle_version`

File: steps/cordova-config/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  opts:
+    is_expand: true
+    is_required: false
+    summary: Modify the attribute `ios-CFBundleVersion` in the config.xml file
+    title: iOS bundle version
+- ios_CFBundleIdentifier: $CORDOVA_IOS_BUNDLEIDENTIFIER
```
Comment (godrei) [inline steps/cordova-config/0.0.1/step.yml]: please use snake_case input keys, like: `ios_bundle_identifier`

**Comment by godrei** [comment]
Hi @blackjid ,
thanks for the PR, i'm happy to see that you solved to modify the `config.xml`.
Please have a look on my review, update your step.yml and share a new version.

File: steps/cordova-config/0.0.1/step.yml (hunk 2 above)
Comment (blackjid) [reply steps/cordova-config/0.0.1/step.yml]: Yeah, you're right. I was trying to stick to the original attribute format, but... is ugly. And people don't need to really know whats happening underneath.

**Pull request closed without merging**